‘sleep_for’ is not a member of ‘std::this_thread’

```
...
[ 50%] Building CXX object CMakeFiles/keyboard_backlight.dir/kbd_backlight.cpp.o
/home/milk/.cache/yay/tp-kb-backlight-git/src/keyboard-backlight/kbd_backlight.cpp: In function ‘void brightness_control(const string&, long unsigned int)’:
/home/milk/.cache/yay/tp-kb-backlight-git/src/keyboard-backlight/kbd_backlight.cpp:248:35: error: ‘sleep_for’ is not a member of ‘std::this_thread’
  248 |                 std::this_thread::sleep_for(sleepTime);
      |                                   ^~~~~~~~~
make[2]: *** [CMakeFiles/keyboard_backlight.dir/build.make:76: CMakeFiles/keyboard_backlight.dir/kbd_backlight.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:137: CMakeFiles/keyboard_backlight.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
==> ERROR: A failure occurred in build().
...
```

--- a/kbd_backlight.cpp
+++ b/kbd_backlight.cpp
@@ -42,6 +42,7 @@
 #include <future>
 #include <csignal>
 #include <fstream>
+#include <thread>
 
 using namespace std::chrono_literals;
 

